sign_oca/controllers/main.py

```diff
@@ -104,3 +104,62 @@ def get_sign_oca_sign_access(
         return signer_sudo.action_sign(
             items, access_token=access_token, latitude=latitude, longitude=longitude
         )
+
+    def get_sign_requests_domain(self, request):
+        domain = [
+            ("request_id.state", "=", "sent"),
+            ("partner_id", "child_of", [request.env.user.partner_id.id]),
+            ("signed_on", "=", False),
+        ]
+        return domain
+
+    def _prepare_sign_portal_rendering_values(self, page=1, sign_page=False, **kwargs):
+        SignRequests = request.env["sign.oca.request.signer"].sudo()
+        domain = self.get_sign_requests_domain(request)
+
+        self._items_per_page = 10
```

Comment:
```suggestion
        self._items_per_page = 10
```
This variable should be declared outside the function. Or it is possible to just declare the local variable type in the current function.

Reply:
I believe that this should not be modified and the value should be left as it is already defined,

Reply:
This definition is not necessary (`sale` for example does not define it), when it is not defined the value of the `portal` module (https://github.com/odoo/odoo/blob/4cb362fcc3c2ba2489ef1de7c20c9a4fd8921ab8/addons/portal/controllers/portal.py#L133) will be used, something that in most cases is enough. It will only be re-defined when you want to change the “_default_” value (`hr_timesheet` for example), but I think `sign_oca` does not need to change it.